src/server/game/Server/Packets/LFGPackets.cpp

```diff
@@ -36,3 +36,354 @@ ByteBuffer& operator<<(ByteBuffer& data, WorldPackets::LFG::RideTicket const& ti
 
     return data;
 }
+
+ByteBuffer& operator>>(ByteBuffer& data, WorldPackets::LFG::LfgBootInfo& bootInfo)
+{
+    data >> bootInfo.VoteInProgress;
```

Comment:
No that I'm not completely incoherent, I realize what you mean. I will push an update soon for it. Don't need that method at all.